Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Image for building packages in docker
+FROM golang:1.13-alpine as builder
+
+RUN apk --no-cache add ruby ruby-dev ruby-etc make gcc g++ rpm git tar && \
+    gem install --no-user-install --no-document fpm
+
+
+
+FROM builder AS build
+COPY . ./graphite-ch-optimizer
+
+RUN cd graphite-ch-optimizer && \
+    make -e CGO_ENABLED=0 build
+
+
+
+FROM alpine:latest
+
+RUN apk --no-cache add ca-certificates tzdata && mkdir /graphite-ch-optimizer
```

Comment:
do we really need those packages? if there is no way without them you cannot use `FROM scratch` unfortunately. but maybe there is a way